### PR TITLE
sphinx 3.4.1 requires alabaster

### PR DESCRIPTION
**Pull request by spirillen**
We are missing alabasta to satisfy the sphinx doc build

```python
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
sphinx 3.4.1 requires alabaster<0.8,>=0.7, which is not installed.
```

### CODE REVIEW

**Comment by spirillen** [comment]
:laughing: 

Which one should we choose, 1.4 is coming :+1: 

```
<<<<<<< patch-4
SQLAlchemy<1.4,>=1.3
=======
SQLAlchemy>=1.3.20
>>>>>>> dev
```

**Comment by funilrys** [comment]
None of them. Look at the patch. https://github.com/funilrys/PyFunceble/commit/2a7bccbf64813db811505bac53e39ae630cf26bc

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+alabaster<0.8,>=0.7
```
Comment (funilrys) [inline requirements.txt:1]: I don't feel comfortable maintaining this. We had the same problem with `urllib3` in the past. I don't want to do the same mistake again.

**Review by funilrys — CHANGES_REQUESTED** [review]
What I mean: It is not a direct dependency on anything PyFunceble (the tool) directly needs.  Please create another `requirements.docs.txt` file just for everything related to the documentation.

Thank you for your time @spirillen!

File: requirements.txt (hunk 1 above)
Comment (spirillen) [reply requirements.txt:1]: > None of them. Look at the patch. [2a7bccb](https://github.com/funilrys/PyFunceble/commit/2a7bccbf64813db811505bac53e39ae630cf26bc)
> 
> In other words: I handle both cases.

Saw that one after committing :smiley:

**Comment by spirillen** [comment]
> What I mean: It is not a direct dependency on anything PyFunceble (the tool) directly needs. Please create another `requirements.docs.txt` file just for everything related to the documentation.
> 
> Thank you for your time @spirillen!

As requested :smiley: despite I'm a bit confused about your second comment <https://github.com/funilrys/PyFunceble/pull/174#discussion_r553486570> if you really won't this at all?

**Comment by funilrys** [comment]
> > What I mean: It is not a direct dependency on anything PyFunceble (the tool) directly needs. Please create another `requirements.docs.txt` file just for everything related to the documentation.
> > Thank you for your time @spirillen!
> 
> As requested  despite I'm a bit confused about your second comment [#174 (comment)](https://github.com/funilrys/PyFunceble/pull/174#discussion_r553486570) if you really won't this at all?

My second comment about feeling bad about maintaining such restriction is all about distribution. Imagine: We have millions of downloads, sometimes in a not so good isolated environment. Putting that line in the required file assume that PyFunceble needs it actively. It is not the case.
 It is in fact needed to generate the documentation (and I will update the readme accordingly). 

In the past, I had `urllib3` into the readme and led to #92 and other underlying noise (per email) that I don't want to constantly manage. Therefore, I choose to be very careful about what I write in there now. In the past, I was alone and I could change it as I want but right now it is not that easy anymore. Indeed issues like #173 may happen from time to time because I don't change my development environment every month. But thanks to our workflow (I'm not talking about GHA), such issues never happen to come in the stable version. At least normally. 

It's all about caring about others while changing something in the `requirements.txt` file.

File: requirements.docs.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+alabaster<0.8,>=0.7
```
Comment (funilrys) [inline requirements.docs.txt:1]: Can we now put everything we need for the documentation in there? 😂

**Review by funilrys — COMMENTED** [review]
Awesome thank you.

**Comment by funilrys** [comment]
Thank you @spirillen.
Merging.

**Pull request merged**